gui/src/main.rs

```diff
@@ -119,8 +119,8 @@ impl Application for GUI {
 
     fn title(&self) -> String {
         match self.state {
-            State::Installer(_) => String::from("Liana Installer"),
-            _ => String::from("Liana"),
+            State::Installer(_) => format!("Liana v{}.{} Installer ", VERSION.major, VERSION.minor),
```

Comment:
Is the extra space at the end intentional?

Reply:
(for trigger CI)

Reply:
Ah OK, I guess better to remove it again though?

Reply:
done